pkg/cgroups/cgroupid_linux.go

```diff
@@ -0,0 +1,12 @@
+package cgroups
+
+import "encoding/binary"
```

Comment:
missing import "golang.org/x/sys/unix".

Reply:
Yes, you're right. I was building locally on a `darwin` machine. Let me do so in a container too.